src/main/scala/molmed/config/FileAndProgramResourceConfig.scala

```diff
@@ -47,12 +47,15 @@ trait FileAndProgramResourceConfig {
   @Argument(doc = "snpEff reference to use", fullName = "snpEff_reference", shortName = "snpEffRef", required = false)
   var snpEffReference: String = _
 
+  @Argument(doc = "Indexed reference file for use with novoalignCS", fullName = "novoalignCSReference", shortName = "nvoalnCSRef", required = false)
+  var novoalignCSReference: File = _
+
   /**
    * Paths to programs
    */
 
-  @Input(doc = "The path to the binary of bwa (usually BAM files have already been mapped - but if you want to remap this is the option)", fullName = "path_to_bwa", shortName = "bwa", required = false)
-  var bwaPath: File = _
+  @Input(doc = "The path to the binary of bwa (usually BAM files have already been mapped - but if you want to remap this is the option)", fullName = "path_to_aligner", shortName = "alner", required = false)
+  var alignerPath: File = _
```

Comment:
Might be better to keep this as `bwaPath` since as things are generally setup here they set it as the path to a specific program?